src/components/picker/picker-column.ts

```diff
@@ -404,11 +404,7 @@ export class PickerColumnCmp {
     }
 
     if (emitChange) {
-      if (this.lastIndex === undefined) {
-        // have not set a last index yet
-        this.lastIndex = this.col.selectedIndex;
-
-      } else if (this.lastIndex !== this.col.selectedIndex) {
```

Comment:
`lastIndex` is only used in these three lines of code, and I couldn't find any reason why the first event shouldn't be emitted to the parent component.